src/plugin/custom/summernote-custom-font.js

```diff
@@ -0,0 +1,142 @@
+                        }
+                    });
+                }
+
+                $customFontBtn = ui.buttonGroup([
+                    ui.button({
+                        className: 'dropdown-toggle',
+                        contents: ui.dropdownButtonContents(
+                            '<span class="note-current-fontname"></span>', options
+                        ),
+                        tooltip: lang.customFont.name,
+                        data: {
+                            toggle: 'dropdown',
+                        },
+                    }),
+                    ui.dropdownCheck({
+                        className: 'dropdown-fontname',
+                        checkClassName: options.icons.menuCheck,
+                        items: fontNames,
+                        title: lang.customFont.name,
+                        template: (item) => {
+                            if (typeof item === 'object') {
+                                return '<span style="font-family: ' + item.value + '">' + item.name + '</span>';
+                            } else if (typeof item === 'string') {
+                                return '<span style="font-family: ' + item + '">' + item + '</span>';
+                            }
+                        },
+                        click: function (event) {
+                            context.createInvokeHandler('customFont.setFontName')(event)
+                            context.invoke('customFont.updateCurrentStyle')
+                        }
+                    }),
+                ]).render();
+                return $customFontBtn
+            })
+
+            this.setFontName = function (value) {
+                context.invoke("beforeCommand");
+                modules.editor.fontStyling.call(modules.editor, 'font-family', value)
+                context.invoke("afterCommand");
+            }
+
+            this.updateCurrentStyle = function(){
+                const styleInfo = context.invoke('editor.currentStyle');
+                if (styleInfo['font-family']) {
+                    const fontNames = styleInfo['font-family'].split(',').map((name) => {
+                        return name.replace(/[\'\"]/g, '')
+                        .replace(/\s+$/, '')
+                        .replace(/^\s+/, '');
+                    });
+                    const showFontName = Array.isArray(fontNames) ? fontNames[0] : fontNames
+
+                    $customFontBtn.find('.dropdown-fontname a').each((idx, item) => {
+                        const $item = $(item);
+                        const showFontName = typeof fontNames === 'object' ? fontNames[0] : fontNames 
+                        const isChecked = ($item.data('value') + '') === (showFontName + '');
+                        $item.toggleClass('checked', isChecked);
+                    });
+                    $customFontBtn.find('.note-current-fontname').text(showFontName).css('font-family', showFontName);
```

Comment:
根據儲存的 `updateFontName` 更新 button

```suggestion
                    $customFontBtn.find('.note-current-fontname').text(updateFontName.name).css('font-family', updateFontName.value);
```

Reply:
Fixed at https://github.com/iq-service-inc/react-summernote/pull/3/commits/476fc07e4bd56ae330ba5c21c1364510d161cfeb